cmd/build/command.go

```diff
@@ -138,6 +139,9 @@ func (bc *Command) getBuilder(options *types.BuildOptions) (Builder, error) {
 		}
 	}
 
+	// TODO: check if global registry is active
+	cache.AddDefaultPullCache(manifest)
```

Comment:
As we talked offline, the okteto image can be empty at this point

Reply:
I moved the logic 👍